dockers/docker-orchagent/ndppd.conf.j2

```diff
@@ -0,0 +1,38 @@
+{% block banner %}
```

Comment:
can we have a test to validate the configuration? i think there is some example in sonic-cfggen test cases.

Reply:
Added test